### PR TITLE
Packager refactor

### PR DESCRIPTION
**Pull request by joshuaclayton**

### CODE REVIEW

File: lib/kumade/packagers/jammit_packager.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+end
+
+module Kumade
+  class JammitPackager
+    def self.assets_path
+      File.join(Jammit::PUBLIC_ROOT, Jammit.package_path)
+    end
+
+    def self.installed?
+      !!defined?(Jammit)
```
Comment (qrush) [inline lib/kumade/packagers/jammit_packager.rb:13]: gross, why `!!` ?

File: lib/kumade/packagers/jammit_packager.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+begin
+  require "jammit"
+rescue LoadError
+end
+
+module Kumade
+  class JammitPackager
+    def self.assets_path
+      File.join(Jammit::PUBLIC_ROOT, Jammit.package_path)
```
Comment (qrush) [inline lib/kumade/packagers/jammit_packager.rb:9]: Any reason why `PUBLIC_ROOT` isn't a `Pathname` ?

File: lib/kumade/packagers/jammit_packager.rb (hunk 1 above)
Comment (joshuaclayton) [inline lib/kumade/packagers/jammit_packager.rb:13]: i want it to return a boolean.

``` ruby
>> defined?(FuckYeahNickQuaranto)
=> nil
>> defined?(FuckYeahJoshClayton)
=> "constant"
```

File: lib/kumade/packagers/jammit_packager.rb (hunk 2 above)
Comment (joshuaclayton) [inline lib/kumade/packagers/jammit_packager.rb:9]: oh, does Jammit do that? that'd be pretty nice.

Comment (qrush) [inline lib/kumade/packagers/jammit_packager.rb:9]: I can't read, we should just leave it...jammit probably doesnt have one

File: lib/kumade/packagers/jammit_packager.rb (hunk 1 above)
Comment (qrush) [inline lib/kumade/packagers/jammit_packager.rb:13]: hurray, a core ruby ? method that returns non booleans

Comment (joshuaclayton) [inline lib/kumade/packagers/jammit_packager.rb:13]: yeah... believe me, i tried to avoid it.

**Pull request merged**